pkg/util/kbutil/kbutil.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 The Kubeflow Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kbutil
+
+import (
+	kubebatchclient "github.com/kubernetes-sigs/kube-batch/pkg/client/clientset/versioned"
+)
+
+var kubeBatchClientSet kubebatchclient.Interface
+
+//SetKubeBatchClientSet sets kube-batch client set.
+func SetKubeBatchClientInterface(clientset kubebatchclient.Interface) {
```

Comment:
Those helper func seems unnecessary.

Reply:
we need kube-batch client interface to create and delete podGroup, but if we add it in TFController, then APIs will change, that is why, in util kube-batch util is created and we are setting and getting kube-batch client interface

Reply:
which APIs are you referring to?

Reply:
APIs for creation and deletion of PodGroup which has been implemented in kube-batch

Reply:
Since it is a common feature across operators, you may keep client interface in https://github.com/kubeflow/tf-operator/blob/master/pkg/common/jobcontroller/jobcontroller.go#L80

Reply:
But this change might affect v1beta1 also, since APIs are same for both v1beta1 and v1beta2. Since we change JobController API used for creating new JobController will change, so to accomadate this change we have to change to change function calls in v1beta1 also. If that is okay, I can make those changes.

Reply:
@thandayuthapani  Let's keep the client interface in jobcontroller and make the necessary changes.

Reply:
Since it is a common feature, we can keep it in JobController and changes can be made for both versions.